### PR TITLE
[docs] Update the link from https://github.com/apple/ to https://github.com/swiftlang/

### PR DESCRIPTION
**Pull request by lamtrinhdev**
### PR Summary:
PR simply fixes the link from:

+ "https://github.com/apple/swift/tree/main/docs/HowToGuides/FAQ.md" => "https://github.com/swiftlang/swift/tree/main/docs/HowToGuides/FAQ.md

### CODE REVIEW

**Comment by lamtrinhdev** [comment]
@swift-ci please smoke test

**Comment by AnthonyLatsis** [comment]
@lamtrinhdev Could you compose all of your updates to links in the `docs` directory in a single PR instead of creating separate PRs per file? Merging this as a single commit is a lot cleaner.

**Comment by lamtrinhdev** [comment]
Dear @AnthonyLatsis , 

Sure. Let me update all of them in the docs folder.

Thanks,
Lam

**Comment by lamtrinhdev** [comment]
Dear @AnthonyLatsis ,

Thank you for the suggestion. I just updated all the links from https://github.com/apple/swift to https://github.com/swiftlang/swift Please help me to review it.

Thanks,
Lam

**Comment by ktoso** [comment]
AFAICS LGTM, but we need to squash when merging this, or you should squash yourself.

**Comment by ktoso** [comment]
@swift-ci please smoke test

**Comment by lamtrinhdev** [comment]
Thank you for your feedback @ktoso .

**Comment by lamtrinhdev** [comment]
Dear @AnthonyLatsis ,

I just completed all my update. If you have time. Please support me to review the PR.

Thanks,
Lam

**Comment by ktoso** [comment]
@swift-ci please smoke test

**Review by AnthonyLatsis — CHANGES_REQUESTED** [review]

File: docs/AndroidBuild.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Windows.
 
 ## 1. Clone the repositories
 1. Configure git to work with Unix file endings
-1. Clone `apple/swift-llvm` into a directory named `llvm`
+1. Clone `swiftlang/swift-llvm` into a directory named `llvm`
```
[the remark this answers is not in the document]
Comment (lamtrinhdev) [reply docs/AndroidBuild.md]: Thank you for your support @AnthonyLatsis . I updated it.

File: docs/AndroidBuild.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ subst S: <path to sources>
 
 ```cmd
 S:
-git clone https://github.com/apple/swift-llvm llvm
+git clone https://github.com/swiftlang/swift-llvm llvm
```
[the remark this answers is not in the document]
Comment (lamtrinhdev) [reply docs/AndroidBuild.md]: Thank you for your support @AnthonyLatsis . I updated it.

**Comment by lamtrinhdev** [comment]
@swift-ci please smoke test

**Comment by AnthonyLatsis** [comment]
> @swift-ci please smoke test

FWIW, this will not work. You need permissions.

**Review by AnthonyLatsis — APPROVED** [review]

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Comment by AnthonyLatsis** [comment]
Thank you for helping with the migration, Lam.

**Pull request merged**